op-guide/docker-deployment.md

```diff
@@ -1,14 +1,13 @@
 # TiDB Docker Deployment 
 
-To learn the TiDB architecture, please see [TiDB Architecture](../README.md#TiDB-Architecture).
+To learn the TiDB architecture, see [TiDB Architecture](../README.md#TiDB-Architecture).
 
 ## Preparation
 
-## Preparation
-Before you start, make sure that:
+Before you start, make sure that you have:
 
-+ Installed the latest version of [Docker](https://www.docker.com/products/docker) 
-+ Pulled the TiDB/TiKV/PD docker images from PingCAP's Docker Hub repositories
++ Install the latest version of [Docker](https://www.docker.com/products/docker) 
```

Comment:
Install -> Installed because the leading sentence is “make sure that you have”.  If it is "make sure that", you can use Install.